content/asset-tracking/index.textile

```diff
@@ -0,0 +1,157 @@
+
+* Passing data to a third-party service.
+* Persistence of data to a database for later retrieval.
+
+As the asset being tracked moves, it publishes its position through the app to an Ably channel. Any subscribers to the channel can then receive positional information, which can be displayed on a detailed map.
+
+The following diagram provides an overview of an asset tracking use case:
+
+<a href="/images/screenshots/asset-tracking/overview.png" target="_blank">
+  <img src="/images/screenshots/asset-tracking/overview.png" style="width: 100%" alt="Overview">
+</a>
+
+In a typical asset tracking scenario the key participants are:
+
+- asset (trackable) := Item that the customer wants to track such as driver, vehicle, container, and so on. This also known as a @Trackable@ in the SDKs.
+- driver := Person driving the asset physically, or entity controlling the asset programatically.
+- customer := Entity that uses the Ably Asset Tracking SDKs to build their product to solve the asset tracking problem for their users.
+- end user := Person that uses the product built by the customer and uses the asset tracking capabilities.
+
+Each asset being tracked must have a @trackableId@, and this must be globally unique within an Ably asset tracking account.
+
+The Ably Asset Tracking SDKs also allow for various battery optimization scenarios, so frequency of updates can be modified based on various parameters such as distance to destination, subscriber status, map zoom level, and existing battery levels.
+
+h2(#battery-optimization). Battery optimization scenarios
+
+One of the biggest challenges in asset tracking is combining high resolution of updates with low battery consumption on the asset tracking device. For example, in a delivery tracking use case you may want your users to have a smooth and realtime visualization of where their delivery is on the map, but at the same time sending realtime locations from the driver's device frequently will rapidly drain its battery.
+
+There are two key factors that impact asset tracking device battery usage:
+
+1. GPS sensor - obtaining location of the asset.
+2. Networking - sending location updates over network.
+
+It is hard to reduce the battery consumption of the GPS sensor in some situations, such as when the device is being used for navigation at the same time as tracking an asset. However, it is possible to adjust the networking aspect of resolution.
+
+The following scenarios are examples of how update resolution can be changed depending on the state of the asset and subscribers:
+
+* Updates can be sent to the subscriber more often when asset is closer to destination.
+* Updates don't need to happen as often when the user of the subscribing SDK is not observing the location of the asset, for example, the user of delivery app is not checking the app for the driver's location.
```

Comment:
"don't need to be sent" instead of "don't need to happen"

Reply:
I've rewritten all those bullet points.